subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -5833,6 +5890,9 @@ static void socket_loop(void)
 		}
 
 		for (i = 0; i < sock_nfds; i++) {
+
+			if (sock_ctx[i]->sock_fd < 0)
```

Comment:
Was this change intended? It seems a bit off.

Reply:
yes true. I need to check. I think there should be continue inside.